### DISCUSSION TITLE
How to show AM-PM formatted time picker for date range picker

### DISCUSSION THREAD

**Discussion by rani-agrawal12** [rsuite/rsuite#2160] [Q&A] [answered]
I want date range picker in "dd-MM-yyyy hh:mm aa" format. when I have used below code for the date range picker. still the time range picker coming in 24 hour format.
<DateRangePicker placeholder="Select Date Range" format="dd-MM-yyyy hh:mm aa" showMeridian={true} />

This is exactly what I'm looking for date range picker.
![screenshot1](https://user-images.githubusercontent.com/72008999/143459071-bc5b61fc-ba2a-47a0-93d6-f2bf9c554cbf.png)

This is what i got.
![screenShot_2](https://user-images.githubusercontent.com/72008999/143462152-71627ca6-f987-4ae5-9bf0-32fef770c5d2.png)

**Comment by SevenOutman** [comment]
Hi @rani-agrawal12, looks like we missed this case. I'm creating an issue from this discussion to track the fixing of this problem.

**Comment by simonguo** [comment, marked as answer]
fixed in [5.2.4](https://github.com/rsuite/rsuite/releases/tag/5.2.4)
